guides/deploy/README.md

```diff
@@ -37,10 +37,10 @@ Verify you have correct `$AKASH_NODE`, that you have populated while [configurin
 ```sh
 echo $AKASH_NODE $AKASH_CHAIN_ID
 
-tcp://rpc-edgenet.akashdev.net:2665 akash-edgenet-1
+http://147.75.195.69:26657 edgenet-4
```

Comment:
These shouldn't really need to change because it's just an example.  But okay either way.